lib/KCard/index.vue

```diff
@@ -167,6 +172,19 @@
         type: String,
         default: 'centerInside',
       },
+      /**
+       * Controls the alignment of the thumbnail area in horizontal layouts.
+       * Only applies to horizontal layouts with 'small' or 'large' thumbnail display.
+       * Ignored in other layouts.
+       * @type {String}
+       * @values 'left', 'right'
+       * @default 'left'
+       */
+      thumbnailAlign: {
+        type: String,
+        default: thumbnailAlign.LEFT,
```

Comment:
We need to hardcode `default: 'left'` here so the documentation shows the correct value:

![docs](https://github.com/user-attachments/assets/6bd8a503-8ac7-4751-964f-6ee1ce7703b3)